cmd/minikube/cmd/cache.go

```diff
@@ -25,6 +25,8 @@ import (
 	"k8s.io/minikube/pkg/minikube/machine"
 )
 
+const cache = "cache"
```

Comment:
I can't figure out what this constant is. Can you give it a clearer name and a comment describing what it's there for?

Update: I see from later that it's used for config maps. What do you think about something like:

```
# cacheImageConfigKey is the config field name used to store which images we have previously cached
const cacheImageConfigKey = 'cache'
```

I'm not normally a fan of long variable names, but a good rule of thumb is: the further away a variable is defined from it's usage, the longer it should be, as more context is required. consts are the perfect place to use longer names for clarity.